File: TT02Q03Cometa/Cometa.java
class Cometa{
	public static void main(String[] args){
		int anoAtual = MyIO.readInt();
	
		MyIO.println(Cometa(anoAtual));
	}

	static int Cometa(int anoAtual){
	//o primeiro ano do cometa foi 10
	int n = ((anoAtual - 10)/76)+1;
	return (n*76+10);	
	}

}

File: TT02Q03CometaEmC/Cometa.c
#include <stdio.h>

int cometa(int ano){
	int n = ((ano-10)/76)+1;
  int prox = (n*76+10);
	printf("%d",prox);
  return 0;
}

int main(){
	int ano=0;
	scanf("%d",&ano);

	cometa(ano);
}